### ISSUE TITLE
NVDA doesn't announce the context menus in the downloads window of microsoft edge

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#15055]
### Steps to reproduce:
1. open microsoft Edge
2. Press ctrl+j to go to downloads
3. Get the focus to a download
4. Press the applications key
5. NVDA will not read the options such as open, delete, show in folder etc.
### Actual behavior:

NVDA will not read any options in the menu
### Expected behavior:

NVDA should read the items in the menu as arrowed up and down.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.1
#### Windows version:
Windows 11, latest build
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge version 114.0.1823.58 
#### Other information about your system:
Os and edge language is set to Russian, NVDA is set to use the system default language
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by enessaribas** [comment]
Hi Fatih,
This is already reported in this issue.
https://github.com/nvaccess/nvda/issues/14916#issuecomment-1567151187

**Comment by seanbudd** [comment]
Closing as duplicate
